test/jdk/tools/jmod/hashes/HashesTest.java

```diff
@@ -86,6 +86,12 @@
     private final Path lib;
     private final ModuleInfoMaker builder;
 
+    public HashesTest() {
+        mods= null;
+        srcDir=null;
+        lib= null;
+        builder=null;
+    }
```

Comment:
This looks like a workaround. Can you instead see if the fields can be changed to non-final and place the constructor with a method that has the `@BeforeClass` annotation?

Reply:
I can look to update the test.  As I mentioned in my reply to Chris, I had thought about introducing an inner class bug decided to go the route of the default/no-arg constructor as it required fewer changes.